akka-docs/src/main/paradox/typed/testing.md

```diff
@@ -212,16 +215,19 @@ a dependency on JUnit to use this.
 
 @@@ div { .group-scala } 
 
-It often makes sense to introduce a common base class for all tests using the async test kit, here is an example how to
-hook it into a ScalaTest test suite.
+If you are using ScalaTest you can extend `akka.actor.testkit.typed.scaladsl.ActorTestKitWordSpec` to
+have the async test kit automatically shutdown when the test is complete.
+
+Note that the dependency on ScalaTest is marked as optional from the test kit module, so your project must explicitly include
+a dependency on ScalaTest to use this.
```

Comment:
👍

Reply:
Was thinking if we should include which version at least... but that will be a hassle to maintain so nah